packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -132,6 +124,25 @@ export class VaTable {
     this.sortAscending = !this.sortAscending;
   }
 
+  private handleSlotChange(e): void {
+    const rows = e.target.assignedElements();
+
+    rows.forEach(row => {
+      const cells = elementChildren(row);
+
+      cells.forEach((cell: HTMLSpanElement, colNum) => {
+        if (isNumeric(cell.textContent)) {
+          cell.classList.add('medium-screen:vads-u-text-align--right');
```

Comment:
Duplicating this isn't great because it open up the door for a cell and header on the same column to have different alignment. The `alignment` object existed to ensure that cells and headers had the same alignment.

Reply:
Yeah the problem I have run into is that `handleSlotChange` is triggered on first load of the component, but it is fired before `componentDidLoad`, so even if I moved `alignment` like I did to the `columns` array, it would not be populated when `handleSlotChange` is first ran. It may be possible to make a separate handler for the `onSlotChange` of the 'headers' slot that would populate both `alignment` and `columns`, but I'm not sure if there is a possibility of there being a timing issue there.

Reply:
Having a `slotchange` handler to handle the pagination use case does seem like the right direction, but I haven't dug deeper into this to see what other changes would need to happen.

Reply:
@bkjohnson do you think this is something that warrants a ticket so we have dedicated time to figure out a solution and test it?